app/src/main/res/layout/admin_pin_activity.xml

```diff
@@ -57,35 +56,53 @@
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@+id/admin_pin_main_text" />
 
-      <org.oppia.android.app.profile.ProfileInputView
+      <com.google.android.material.textfield.TextInputLayout
         android:id="@+id/input_pin"
```

Comment:
update id with more understandable naming inclined towards the screen. 
for reference from other file `admin_auth_input_pin`

Reply:
Does `admin_pin_auth_pin` sound good?

Reply:
`admin_pin_input_pin` is already taken in `pin_password_activity`. Can you please suggest some other name?

Reply:
Done